.github/CODEOWNERS

```
@@ -4,9 +4,64 @@
 #
 # https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
 
+# BRE for publish workflow changes
+.github/workflows/publish-*.yml @bitwarden/dept-bre
+
+# Shared workflows ownership
+
+## BRE & SM teams shared ownership
+.github/workflows/build-cli-docker.yml @bitwarden/dept-bre
```

I think it can be simplified to one line:
```suggestion
.github/workflows/build-cli-docker.yml @bitwarden/dept-bre @bitwarden/team-secrets-manager-dev
```